test/integration/targets/ec2_group/tasks/main.yml

```diff
@@ -804,7 +1049,47 @@
           - 'not result.tags'
 
     # ============================================================
+    - name: test adding a rule and egress rule descriptions (expected changed=true) (CHECK MODE)
```

Comment:
This doesn't fail but the output looks wrong to me.
```
changed: [localhost] => {
    "changed": true,
    "description": "Created by ansible integration tests",
    "group_id": "sg-0e71e747",
    "group_name": "ansible-test-macbook-pro-63205108",
    "invocation": {
        "module_args": {
            "aws_access_key": "ABIAJT7AJOLXTL7IVL5A",
            "aws_secret_key": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "description": "Created by ansible integration tests",
            "ec2_url": null,
            "group_id": null,
            "name": "ansible-test-macbook-pro-63205108",
            "profile": null,
            "purge_rules": true,
            "purge_rules_egress": true,
            "purge_tags": true,
            "region": "us-east-1",
            "rules": [
                {
                    "cidr_ipv6": "1001:d00::/24",
                    "ports": [
                        8281
                    ],
                    "proto": "tcp",
                    "rule_desc": "ipv6 rule desc 1"
                }
            ],
            "rules_egress": [
                {
                    "cidr_ip": "2.2.2.2/32",
                    "ports": [
                        8282
                    ],
                    "proto": "tcp",
                    "rule_desc": "egress rule desc 1"
                }
            ],
            "security_token": "",
            "state": "present",
            "tags": null,
            "validate_certs": true,
            "vpc_id": "vpc-52baff29"
        }
    },
    "ip_permissions": [
        {
            "from_port": 8182,
            "ip_protocol": "tcp",
            "ip_ranges": [
                {
                    "cidr_ip": "1.1.1.1/32"
                }
            ],
            "ipv6_ranges": [],
            "prefix_list_ids": [],
            "to_port": 8182,
            "user_id_group_pairs": []
        }
    ],
    "ip_permissions_egress": [
        {
            "ip_protocol": "-1",
            "ip_ranges": [
                {
                    "cidr_ip": "0.0.0.0/0"
                }
            ],
            "ipv6_ranges": [],
            "prefix_list_ids": [],
            "user_id_group_pairs": []
        }
    ],
    "owner_id": "448830907657",
    "tags": {},
    "vpc_id": "vpc-52baff29"
}
```
The new ingress and egress rule have not replaced the previous ones.